client/src/components/products/AddProduct.js

```diff
@@ -78,7 +83,8 @@ export default class AddProduct extends Component {
           },
           availability: null,
           warrantyUntil: null,
-          condition: ""
+          condition: "",
+          createdAt: new Date()
```

Comment:
this needs to be changed to match with the initial state as 'null'

Reply:
True that. Thanks for the review, I updated the PR. 🎈